R/commonPowerPlotting.R

```diff
@@ -0,0 +1,315 @@
+# ==== Default Plotting Configurations ====
+
+# Common theme to be used for all plots
+.pwrPlotTheme <- function() {
```

Comment:
> I think themeJasp itself actually deprecated? 

You are right!

In this case I would compare to

```r
plotObject +
  jaspGraphs::geom_rangeframe() +
  jaspGraphs::themeJaspRaw(legend.position = "right")
```

The idea is to have a uniform style of plots across JASP, so it makes sense to stick to that. Unless there is a reason why the default JASP style does not work for the particular plot (that does not seem the case here but you can convince me otherwise).

> If this is important, I guess we could just quickly change it over as this is handled in the function anyways. 🤷‍♂️

Yes, it's nice that you wrapped it in a separate function!

An alternative is to let the user switch between different themes (this is for example done in Mixed Models module) if you think it's worth for people to try different looks. But usually it's not worth the effort.